src/components/SubmitResponsePage.jsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
-import Alert from "./Alert";
 import FormContainer from "./FormContainer";
 import { Formik, Form } from "formik";
 import { HasResponseErrors } from "../utilities/CitysourcedResponseHelpers";
-import ButtonDisplay from "./buttonDisplay";
+import { Alert } from "@baltimorecounty/dotgov-components";
+import SeButton from "./SeButton";
 import { GoHome } from "../Routing";
 
 const successBodyContent = (
-  <React.Fragment>
+  <Alert className="status" type="success" icon="far fa-check">
```

Comment:
If i'm following this should use the page validation component? (which maybe it doesn't after our conversation).

Reply:
No. Page validations are only on elements that arent filled in or incorrectly filled in. They arent alerts like these. Alerts coming from the library are very different from page validations. 

![image](https://user-images.githubusercontent.com/40430541/80527375-45426180-8962-11ea-9008-38e6f6cd3c51.png)

Reply:
Having a compent called Alerts coming in from the library and an Alerts component in the application seemed confusing to me but maybe not?

Reply:
could we changed it to something like `ErrorMessage` or something along those lines? because it seems like this is specific to a form field? Reason I'm suggesting Error Message is formik, https://jaredpalmer.com/formik/docs/api/errormessage. Or maybe something like FormError, FieldError?

Reply:
Yeah thats fine. I just didnt think it made sense to keep it named Alert. Im open to any other kind of name.